### PR TITLE
Update README to capture pipeline details

### PR DESCRIPTION
**Pull request by gregwood-db**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,32 +94,133 @@ Support Matrix for Import and Export Operations:
 | Libraries         | Supported    | Unsupported  |
 | Secrets           | Supported    | Supported    |
 | Table ACLs        | Supported    | Supported    |
-| ML Models         | Unsupported  | Unsupported  |
+| ML Models         | Supported    | Supported    |
```
Comment (kevinkim9264) [inline README.md:97]: Can you specify ML models are in alpha phase?

**Review by kevinkim9264 — APPROVED** [review]
thanks!

File: README.md (hunk 1 above)
Comment (gregwood-db) [reply README.md]: @kevinkim9264 done. thanks!

File: README.md [hunk 2]
```diff
@@ -79,32 +94,136 @@ Support Matrix for Import and Export Operations:
...
+
+These can be adjusted per your scenario if needed; in general, if API limits are being hit, you can increase `retry-backoff`, decrease `num-parallel`, or both. 
+
+If script failure occurs, you can safely rerun the same command with --use-checkpoint and --session $SESSION_ID to let the migration pick up from the previous checkpoint and rerun.
+
+#### Updating the AWS Account ID
+If your source and destination workspaces are in different accounts, you will need to update the Instance Profile ARN accordingly during the migration. To do this, run the following command after exporting the workspace assets:
+
+```python3 export_db.py --profile $SRC_PROFILE --update-account-id --use-checkpoint --old-account-id $OLD_AWS_ACCT_ID --new-account-id $NEW_AWS_ACCT_ID --session $SESSION_ID
+```
```
Comment (petrcv-db) [inline README.md:198]: @gregwood-db @kevinkim9264 I don't think this is going to work. 
* `--new-account-id` is not a parameter, it should be `--update-account-id`
* AFAIK `export_db.py` doesn't support sessions. To work around that you need to set something like `--set-export-dir $EXPORT_PATH/$SESSION_ID`. Or is there any other way?
Check out https://github.com/databrickslabs/migrate/issues/162#issuecomment-1102639987
Other than that thanks for the README update!

Comment (gregwood-db) [reply README.md:198]: Thanks @petrcv-db - updated to reflect this.

**Pull request merged**